e2e/cypress/integration/system_console/revoke_all_sessions.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const buttonTheme = {backgroundColor: 'rgba(0, 0, 0, 0.7)', color: 'rgb(255, 255, 255)'};
+
+describe('System Console', () => {
+    it('SC17020 - Revoke All Sessions from System Console', () => {
+        // # Login as System Admin
+        cy.apiLogin('sysadmin').as('sysadmin');
```

Comment:
Doesn't look like you are using the alias you set, so you can probably remove it.

Reply:
I forgot to remove it. It was earlier used, but I had removed those steps. Thanks for pointing it out. It's fixed now.